blog/nodejs-corepack.md

```diff
@@ -0,0 +1,96 @@
+# Corepack the Node.js' manager of package managers
+
+Corepack is the new built-in tool to prepare the required package manager for our projects. We're going to see what comes with this new package for packagers. 📦
+
+## Installation
+
+We're in track to get the new Node.js LTS v16 at the end of October, without a fancy name assigned yet, and we'll get [Corepack](https://github.com/nodejs/corepack) preinstalled in the default Node.js configuration since the [v16.9.0](https://github.com/nodejs/node/releases/tag/v16.9.0). 👌
+
+If we want this tool for previous versions we just need to install it as a global package:
+
+```bash
+$ npm install -g corepack
+```
+
+🧠 Remember that until now, the Corepack v0.9.0 only supports `pnpm`, `npm` and `yarn`.
+
+## Basic usage
+
+We can use use the package manager that we always use in any machine without worry if it is installed or not using by example:
+
+```bash
+$ corepack pnpm install
+```
+
+Corepack will take care that the required manager is ready to process your request.
+
+### Specific versions
+
+The included `npm` version with Node.js LTS v16 is v7, so what if we need `npm@6` for one project and `npm@7` for other?
+
+Well, you will find `corepack` very similar to `nvm` in this scenario.
+
+```bash
+$ corepack prepare npm@6.14.15 --activate
+$ node -v
+v16.9.0
+$ npm -v
+7.21.1
+$ corepack npm -v
+6.14.15
+```
+
+🤯 So, with `corepack prepare` you'll setup `npm@6`, and you can use it every time you need just prefixing your `npm` usage with Corepack, e.g. `corepack npm ci`.
+
+And your global `npm` command will continue untouched being `npm@7`.
+
+🤔 But what is that `--activate` flag? Corepack will keep a cache of used versions of a packager, so you should activate which version you want to use, otherwise you're still using `npm@7` when you use `corepack npm -v`.
```

Comment:
```suggestion
🤔 But what is that `--activate` flag? Corepack will keep a cache of used versions of a packager, so you should activate the specific version that you want to use; otherwise you will still be using `npm@7` when you use `corepack npm -v`.
```